### PR TITLE
Userfriendly cheat list

### PR DESCRIPTION
**Pull request by JonathanERC**
I made this .md document with the cheats that work and those that does not work for making easy to people to identify them. Take a look and consider if is good to publish or not. Thank you for your time and all the good things you all share with us.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ You can launch the Configurator app by clicking on the `Configuration` button lo
 ## Tips and Tricks
 
 - In order to reduce occasional stutters in-game, delete both `ux0:data/gtasa/scache_small_low.txt` and `ux0:data/gtasa/scache_small.txt`, then create a copy of the `ux0:data/gtasa/scache.txt` file to have two version of it. (for example `scache(1).txt` so in the end you end up with both `scache.txt` and `scache(1).txt` inside the `ux0:data/gtasa/` folder), then rename `scache.txt` to `scache_small.txt` and `scache(1).txt` to `scache_small_low.txt` . This will however make the loading screen longer since it needs to compile more shaders ahead.
-- You can input PC cheats by pressing L+SELECT to open the on-screen keyboard. See https://gtagmodding.com/sanandreas/cheats/ for the full list of cheats (you can input cheat codes in lowercase as well as uppercase). Note that a few cheats have been removed from the Android version.
+- You can input PC cheats by pressing L+SELECT to open the on-screen keyboard. See https://github.com/JonathanERC/gtasa_vita/blob/master/cheatList.md for available and unavailable cheats (you can input cheat codes in lowercase as well as uppercase).
```
Comment (TheOfficialFloW) [inline README.md:54]: Point to https://github.com/TheOfficialFloW/gtasa_vita.

**Comment by TheOfficialFloW** [comment]
Thanks. Can you call the file CHEATS.md?

File: README.md (hunk 1 above)
Comment (JonathanERC) [reply README.md]: Sorry, my bad. Now it points to https://github.com/TheOfficialFloW/gtasa_vita/CHEATS.md

**Comment by JonathanERC** [comment]
> 
> 
> Thanks. Can you call the file CHEATS.md?

Of course!

**Review by JonathanERC — COMMENTED** [review]
Now cheats list points to: https://github.com/TheOfficialFloW/gtasa_vita/CHEATS.md

And cheatList.md renamed to CHEATS.md

**Pull request merged**